AlphaWallet/Configuration.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  Configuration.swift
+//  AlphaWallet
+//
+//  Created by Jerome Chan on 30/5/22.
+//
+
+import UIKit
+
+fileprivate func colorFrom(trait: UITraitCollection, lightColor: UIColor, darkColor: UIColor) -> UIColor {
+    switch trait.userInterfaceStyle {
+    case .unspecified, .light:
+        return lightColor
+    case .dark:
+        return darkColor
+    }
+}
+
+struct Configuration {
+    struct Color {
+        struct Semantic {
```

Comment:
An additional `Semantic` doesn't seem necessary, unless we are planning on more sibling "layers" soon?